frontend/sw360-portlet/src/main/webapp/html/admin/vendors/view.jsp

```diff
@@ -68,7 +68,7 @@
     AUI().use('liferay-portlet-url', function () {
         var PortletURL = Liferay.PortletURL;
 
-        require(['jquery', 'utils/includes/quickfilter', 'modules/confirm', /* jquery-plugins: */ 'datatables', 'jquery-confirm'], function($, quickfilter, confirm) {
+        require(['jquery', 'utils/includes/quickfilter', 'modules/confirm', /* jquery-plugins: */ 'datatables.net', 'datatables.net-buttons', 'buttons.print', 'jquery-confirm'], function($, quickfilter, confirm) {
```

Comment:
Why is it necessary to change `'datatables'` to `'datatables.net'`? Please explain.

Reply:
The js for the print functionality itself requires the following scripts
- datatables.net
- datatables.net.buttons
define( ['jquery', 'datatables.net', 'datatables.net-buttons'], function ( $ ))

But I found a solution to use the datatable require statements as before.
We can map the datatables.net name to datable by using the "map" function in our require.js file.

  map: {
            '*': {
                'datatables.net': 'datatables',
                'datatables.net-buttons': 'datatables_buttons'
            }
        }